src/veh_interact.cpp

```diff
@@ -371,6 +378,15 @@ void veh_interact::cache_tool_availability()
                 crafting_inv.has_components( "wheel_bicycle", 1 ) ||
                 crafting_inv.has_components( "wheel_motorbike", 1 ) ||
                 crafting_inv.has_components( "wheel_small", 1 );
+
+    int qual_jack = ceil( double( veh->total_mass() * 1000 ) / TOOL_LIFT_FACTOR );
+    has_jack = g->u.has_quality( "JACK", qual_jack ) ||
+               map_selector( g->u.pos(), PICKUP_RANGE ).has_quality( "JACK", qual_jack ) ||
+               vehicle_selector( g->u.pos(), 1, *veh ).has_quality( "JACK", qual_jack );
```

Comment:
> Inconsistent with crafting and map uses

A lot of stuff uses adjacent tiles and those which don't typically require line-of-sight to the tool, eg. `WELDRIG` etc

> Regardless of the placement, you still need to park pretty exactly next to the vehicle you want to operate on

Yes, but this is intentional. I don't want cranes to become another mandatory item you must craft and bolt onto your vehicle then forget about. There's no point in differing types if their placement isn't significant.

> If you're worried about safety, you can still do it with 1 tile range, by installing the crane 1 tile deep into the vehicle and a door next to it

That's an option but now maneuvering is more difficult. Inside cities space can be especially tight so you'd be passing up on scavenging parts from some military wreckages that you now can't get the angle on.

> because pulling out all the tools would be just too annoying to deal with.

The only difference of cranes having `radius 1` is that you have to successfully maneuver your vehicle to the target. This is very much a feature. The argument about tools is null in that you already have to move your tools every `PICKUP_RANGE` tiles. I can't see any other meaningful differences but I'm open to further discussion.